app/views/samples_reports/show.haml

```diff
@@ -49,6 +49,10 @@
                 .subtitle{id:'print-as-pdf'}
                   = icon_tag "as-pdf", class: "btn-icon icon-text-color"
                   Download PDF
+            .action
+              =button_tag id:'print-as-csv', class: 'subtitle btn-transparent' do
```

Comment:
A DOM ID must be unique on a page. You want a CLASS or a DATA attribute that doesn't have to be unique.

Reply:
But this element is unique in the DOM, indeed there is not other `print-as-csv` elements in the whole project. I'm getting it wrong?